users.md

```diff
@@ -1,40 +1,59 @@
-  * [Geophysical Fluid Dynamics Laboratory](http://www.gfdl.noaa.gov/) - USA
-  * [Meteorological Service Singapore (MSS)](http://www.weather.gov.sg/home/) - Singapore
-  * [Phillippine Atmospheric, Geophysical, and Astronomical Services
-    Administration (PAGASA)](http://bagong.pagasa.dost.gov.ph)  - Phillippines
-  * [South African Weather Service (SAWS)](http://www.weathersa.co.za/) - South Africa
-  * [National Centre for Medium Range Weather
-      Forecasting](http://www.ncmrwf.gov.in/) - India
-  * [Korean Meteorological Administration (KMA)](http://web.kma.go.kr/) - Korea
+* Oceania:
+  * New Zealand:
+    * [National Institute of Water and Atmospheric Research
+      (NIWA)](http://www.niwa.co.nz)
+  * Australia:
+    * [Bureau of Meteorology](http://www.bom.gov.au/)
+    * [ARC Centre of Excellence for Climate Extremes (CLEX)](https://climateextremes.org.au/)
+* Europe:
   * [Centre of Excellence in Simulation of Weather and Climate in
-      Europe](https://verc.enes.org/esiwace/services/sup_cylc) - Europe
-  * [National Center for Atmospheric Reserach (NCAR)](https://ncar.ucar.edu) - USA
-  * [NOAA Environmental Modeling Center](http://www.emc.ncep.noaa.gov/) - USA
-  * [Euro-Mediterranean Center on Climate Change (CMCC)](https://www.cmcc.it) - Italy
-  * [Plymouth Marine Laboratory](https://www.pml.ac.uk/) - UK
-  * [Barcelona Supercomputing Center (BSC)](https://www.bsc.es) - Spain
-  * [National Centre for Atmospheric Science (NCAS)](https://www.ncas.ac.uk) - UK
-  * [Centre for Environmental Data Analysis (CEDA)](http://www.ceda.ac.uk) - UK
-  * [ARC Centre of Excellence for Climate Extremes (CLEX)](https://climateextremes.org.au/) - Australia
-  * [eWaterCycle](http://forecast.ewatercycle.org/) - Netherlands eScience Center, Univerity of Utrecht, TUDelft
+      Europe](https://verc.enes.org/esiwace/services/sup_cylc)
+  * UK:
+    * [Met Office](http://www.metoffice.gov.uk)
+    * [Plymouth Marine Laboratory](https://www.pml.ac.uk/)
+    * [National Centre for Atmospheric Science (NCAS)](https://www.ncas.ac.uk)
+    * [Centre for Environmental Data Analysis (CEDA)](http://www.ceda.ac.uk)
+  * Germany:
+    * [Max-Planck-Institut f&uuml;r
+      Meteorologie](http://www.mpimet.mpg.de/mpimet-startseite/)
+    * [Deutches Klimarechenzentrum (DKRZ)](https://www.dkrz.de/)
+  * Italy:
+    * [Euro-Mediterranean Center on Climate Change (CMCC)](https://www.cmcc.it)
+  * Spain:
+    * [Barcelona Supercomputing Center (BSC)](https://www.bsc.es)
+  * Netherlands:
+    * [eWaterCycle](http://forecast.ewatercycle.org/) at the Netherlands eScience Center, Univerity of Utrecht, TUDelft
+* North America:
+  * USA:
+    * [NRL Marine Meteorology Division](http://www.nrlmry.navy.mil)
+    * [557th Weather Wing](https://www.557weatherwing.af.mil)
+    * [Geophysical Fluid Dynamics Laboratory](http://www.gfdl.noaa.gov/)
+    * [National Center for Atmospheric Reserach (NCAR)](https://ncar.ucar.edu)
```

Comment:
s/Reserach/Research

Reply:
Though this is an old typo and could be fixed separately

Reply:
Old, yes, but I'll fix it here.